templates/publishers/inventory_hosts.yml.j2

```diff
@@ -0,0 +1,21 @@
+---
+# Inventory file
+# Update with your actual hosts and groups
```

Comment:
In the overall flow, how is the user expected to update that file?

Reply:
It makes the flow a little bit more complicated.

We can make it optional in the user, if the user knows the structure beforehand. E.g. already created for other scripts.

Updating the gitrepo is an option but it creates bad UX by adding extra step in an external system with completely different UI (web/cli).

Thanks. Let's keep that on mind.